.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+### Issue details
+
+_Please provide issue details here_.
+
+### Steps to reproduce/test case
+
+_Please provide necessary steps for reproduction of this issue, or better the
+reduced test case (without any external dependencies)_.
+
+### Affected node.js versions
+
+- [ ] master
+- [ ] v5.x
+- [ ] v4.x
+- [ ] v0.12
+- [ ] v0.10
+
+### Affected platforms
+
+- [ ] Linux
+- [ ] Windows
+- [ ] OS X
+- [ ] FreeBSD
+- [ ] Solaris
+- [ ] other _(please specify which)_
+
+### Core subsystem (if known)
+
+- [ ] buffer
+- [ ] child_process
+- [ ] cluster
+- [ ] crypto
+- [ ] dgram
+- [ ] dns
+- [ ] doc
+- [ ] fs
+- [ ] http
+- [ ] https
+- [ ] net
+- [ ] tls
+- [ ] tty
+- [ ] vm
+- [ ] zlib
+- [ ] other _(please specify which)_
```

Comment:
`src` is rather un-usual... I tried to include only popular ones. Also all of them mean both `lib/` an `src/`. Perhaps, I should clarify it?